ext/page_log/palm/palm.c

```diff
@@ -1241,7 +1248,9 @@ palm_terminate(WT_PAGE_LOG *storage, WT_SESSION *session)
     ret = 0;
     palm = (PALM *)storage;
 
-    if (--palm->reference_count != 0)
+    uint32_t new_ref_count = atomic_fetch_sub(&palm->reference_count, 1);
```

Comment:
The name `new_ref_count` is misleading. We are getting the previous value.
```suggestion
    uint32_t old_ref_count = atomic_fetch_sub(&palm->reference_count, 1);
```

Reply:
+1, calling it "new" is misleading.